mava/environment_loop.py

```diff
@@ -631,14 +614,12 @@ def should_run_loop(eval_interval_condition: Tuple) -> bool:
                     # Check for extra logs
                     if hasattr(self._environment, "get_interval_stats"):
                         results.update(self._environment.get_interval_stats())
-                    self._logger.write(results)
-
                 else:
                     result = self.run_episode()
                     episode_count += 1
                     step_count += result["episode_length"]
                     # Log the given results.
-                    self._logger.write(result)
+                self._logger.write(result)
```

Comment:
Similarly here, please keep it at the same indentation level. This is so that we can fall back on the continual logging default when not evaluation interval details have been passed in :)

Reply:
same as above